Makefile

```diff
@@ -176,11 +176,11 @@ azdiskschedulerextender:
 
 .PHONY: container
 container: azuredisk
-	docker build --no-cache -t $(IMAGE_TAG) --build-arg PLUGIN_NAME=${PLUGIN_NAME} -f ./pkg/azurediskplugin/dev.Dockerfile .
+	docker build --no-cache -t $(IMAGE_TAG) --build-arg PLUGIN_NAME=${PLUGIN_NAME} --output=type=docker -f ./pkg/azurediskplugin/Dockerfile .
 
 .PHONY: container-v2
 container-v2: azuredisk-v2
-	docker build --no-cache -t $(IMAGE_TAG) --build-arg PLUGIN_NAME=${PLUGIN_NAME} -f ./pkg/azurediskplugin/dev.Dockerfile .
+	docker build --no-cache -t $(IMAGE_TAG) --build-arg PLUGIN_NAME=${PLUGIN_NAME} --output=type=docker -f ./pkg/azurediskplugin/dev.Dockerfile .
```

Comment:
Shouldn't the V2 build also use `Dockerfile` instead?

Reply:
Kept the dev.Dockerfile for v2 to isolate Andy's upstream changes to Dockerfile and use it for v1 and continue using dev.Dockerfile for v2. Should that be fine?

Reply:
Ah, yes, I understand your reasoning now. However, we don't publish the v1 driver image from Mandalay sources. I'd prefer to maintain a single Dockerfile for the sake of simplicity. To fully separate the two container builds, we would have to make more changes to the Makefile because the `container` and `container-v2` targets aren't used to publish the images consumed by the e2e tests. AFAICT, the `dev.Dockerfile` was intended to be used by developers to build a container image to run locally for development and debugging purposes.

Reply:
Understood. I will continue with a single Dockerfile. Will remove the reference and file dev.Dockerfile.